src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -12,7 +12,15 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String SHOWING_HELP_MESSAGE = "Here are the list of commands available:"
```

Comment:
thanks for the work! perhaps for the commands that require a certain format, the format could be specified? like add command needs n/NAME t/TIME .... etc

Reply:
I agree with Harshita that adding a format will be useful as well so that the user will know how to use the commands _on top of_ just knowing what each command does! Thank you Yue Tong! :)